src/views/users/Register.vue

```diff
@@ -136,13 +144,19 @@ export default {
         /(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}/.test(v) ||
         "Password must contain at least lowercase letter, one number, a special character and one uppercase letter",
     ],
+  
     firstcheckbox: false,
     seccheckbox: false,
   }),
+
   methods: {
     submitForm() {
       this.$refs.form.validate();
     },
+    validatePassword2(value) {
+      return value == this.password || "Passwords don't match."
```

Comment:
Io userei l'operatore `===` al posto di `==` per il solito [problema di tipi del Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness): occorre usare un operatore maggiormente restrittivo in questi casi, come lo [Strict Equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality). Se usi il `===`, ti controlla che il valore del primo operando, ad esempio `"12"`, sia esattamente `"12"`, mentre invece l'operatore `==` potrebbe anche dire che siano uguali i valori `"12"` e `12`. Potrebbe sembrare una piccolezza, ma talvolta potrebbe anche dare comportamenti inaspettati.

Reply:
hai ragione @Daniele-Tentoni , ho modificato